src/Aspire.Hosting/api/Aspire.Hosting.cs

```diff
@@ -1982,16 +1995,28 @@ public void WriteConnectionString(ApplicationModel.IResource resource) { }
         public System.Threading.Tasks.Task WriteEnvironmentVariablesAsync(ApplicationModel.IResource resource) { throw null; }
     }
 
+    [System.Diagnostics.CodeAnalysis.Experimental("ASPIREPUBLISHERS001")]
+    public sealed partial class NullPublishingActivityProgressReporter : IPublishingActivityProgressReporter
```

Comment:
Should this be in Aspire.Hosting or Aspire.Hosting.Testing?

Reply:
That's a good question. I'm not 100% sure.

[NullLogger](https://github.com/dotnet/runtime/blob/main/src/libraries/Microsoft.Extensions.Logging.Abstractions/src/NullLogger.cs) is in Microsoft.Extensions.Logging.Abstractions. Although I don't think we have a logging testing package.

Reply:
@eerhardt any further thoughts